src/Tribe/Views/V2/Hooks.php

```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
 		$assets->disable_v1();
 	}
 
+	public function action_register_widget_styles() {
```

Comment:
:no_entry_sign: Missing doc comment for function action_register_widget_styles()

<sub>hash: 86962694dda0b1b41931f8cc3e439cba</sub>